src/features/message/MessageTokens.jsx

```diff
@@ -41,15 +43,17 @@ const tokenMap = {
 	emoji: Word
 };
 
-const mapToMessage = (token, index) => {
+const mapToMessage = (message, token, index) => {
```

Comment:
Don't love that I have to send `message` here. It's needed to know which message is toggling image visibility. Open to suggestions.